node/lib/TranslatorInfo.ts

```diff
@@ -0,0 +1,26 @@
+
+/**
+ * Information about a translator, as loaded from the translator's manifest.
+ */
+export class TranslatorInfo {
```

Comment:
ah ok.. I see onboardingProperties comment.. looks like it will capture this info.

---
In reply to: [71904732](https://github.com/openT2T/opent2t/pull/1#discussion_r71904732) [](ancestors = 71904732)